src/olympia/devhub/views.py

```diff
@@ -1033,14 +1033,13 @@ def upload_image(request, addon_id, addon, upload_type):
         if is_icon:
             max_size = settings.MAX_ICON_UPLOAD_SIZE
         else:
-            max_size = None
+            max_size = settings.MAX_IMAGE_UPLOAD_SIZE
 
         if max_size and upload_preview.size > max_size:
```

Comment:
nit: we don't need the `max_size and` now `max_size` will always have a non-falsey value.